force-app/main/default/lwc/reviewSessions/reviewSessions.html

```diff
@@ -0,0 +1,44 @@
+<template>
+    <!--<c-section title={labels.reviewSessions}></c-section>-->
```

Comment:
I think it makes sense to have the titles as part of each individual screen, do you want to either use the css here or bring in the section component?

Reply:
@jjbennett , I had to comment this out since I had created this branch from feature/Winter-21 so the `c-section `component does not exist. Once that is merged to the feature branch and when we start working on the back end piece of it we can add it back

Reply:
I think since you will prolly merge first it'd be ok to either copy that section component over from my branch or do a div with the slds for now so your component can be finished. Not a huge deal since we will ultimately have to revisit but it'd be one less thing to deal with later. I will leave it up to you.